lib/sum/test/Sum.cpp

```diff
@@ -0,0 +1,20 @@
+#include "gtest/gtest.h"
+
+#include "Sum.h"
+
+TEST(SumTest, SumNumbersFromFileTest) {
+    EXPECT_EQ(
+        sum_numbers_from_file("files/File1.txt"),
+        4753
+    );
+
+    EXPECT_EQ(
+        sum_numbers_from_file("files/File2.txt"),
+        36
+    );
+
+    EXPECT_EQ(
+        sum_numbers_from_file("files/File3.txt"),
+        -43
+    );
```

Comment:
I think you need to add some test to check the correctness of file data. If a file doesn't contain integer numbers or contains float point numbers.